src/app/pages/ArticlePage/fixtureData.ts

```diff
@@ -1517,10 +1532,39 @@ export const articleDataRussianWithPVButNoWatchMomentsTranslation =
     'Article Headline for Promo in Russian',
     'Article summary in Russian',
     emptyThings,
+
     true,
     plainOptimoBlock,
   ) as unknown as Article;
 
+export const articleDataPidginWithReadTime = articleDataBuilder(
```

Comment:
If we default readTime to something other than 0, then we don't really need this fixture data - we could just use one of the other ones instead.